db/badger.go

```diff
@@ -111,8 +111,8 @@ func (t *badgerTxn) Impl() any {
 }
 
 // NewDb opens a new database at the given path
-func NewDb(path string) (DB, error) {
-	opt := badger.DefaultOptions(path)
+func NewDb(path string, log badger.Logger) (DB, error) {
```

Comment:
Why are we not passing our `utils.Logger`?

Reply:
Because `utils.Logger` includes methods that badger does not require.